Adafruit_SPIDevice.h

```diff
@@ -26,6 +26,14 @@ typedef enum _BitOrder {
   SPI_BITORDER_LSBFIRST = SPI_LSBFIRST,
 } BitOrder;
 
+#elif defined(__ASR6501__)
```

Comment:
please add this to the `defined(ESP32)` line aboe to minimize duplcation

Reply:
Done.